src/intervals/rounding.jl

```diff
@@ -66,75 +103,99 @@ function split_interval_string(T, x::AbstractString)
 function make_interval(::Type{BigFloat}, x::Float64)
     isinf(x) && return Interval(convert(BigFloat,x))
-    make_interval(BigFloat, rationalize(x))  # NB: converts a float to a rational -- this could be slow
+    split_interval_string(BigFloat, string(x))
+end
+make_interval(::Type{BigFloat}, x::Integer) =
+    @thin_round(BigFloat, convert(BigFloat, x))
+make_interval(::Type{BigFloat}, x::BigFloat) = @thin_round(BigFloat, x)  # convert to possibly different BigFloat precision
+function make_interval(::Type{BigFloat}, x::Interval)
+    a = make_interval(BigFloat, x.lo)
+    b = make_interval(BigFloat, x.hi)
+    Interval(a.lo, b.hi)
 end
 
-make_interval(::Type{BigFloat}, x::BigInt)    =  @thin_round(BigFloat, convert(BigFloat, x))
-
-make_interval(::Type{BigFloat}, x::BigFloat)  =  @thin_round(BigFloat, 1.*x)  # convert to possibly different BigFloat precision
-make_interval(::Type{BigFloat}, x::Interval)  =  @round(BigFloat, big(1.*x.lo), big(1.*x.hi))
-
-
-make_interval(::Type{Float64}, x::AbstractString)    =  split_interval_string(Float64, x) #@thin_round(Float64, parse(Float64, x))
-make_interval(::Type{Float64}, x::Irrational) =  make_interval(Float64, make_interval(BigFloat, x))
 
-make_interval(::Type{Float64}, x::Integer)   =  Interval(float(x))    # assumes the int is representable
-make_interval(::Type{Float64}, x::Rational)  =  make_interval(Float64, x.num) / make_interval(Float64, x.den)
+# make_interval for Float64 intervals
+make_interval(::Type{Float64}, x::AbstractString) = split_interval_string(Float64, x)
+function make_interval(::Type{Float64}, x::Irrational)
+    a = with_bigfloat_precision(53) do
+        make_interval(BigFloat,x)
+    end
+    float(a)
+end
+function make_interval(::Type{Float64}, x::Rational)
+    isinf(x) && return Interval(convert(Float64,x))
+    # Interval(float(x.num)) / Interval(float(x.den))
+    @controlled_round(Float64, x.num/x.den, x.num/x.den)
+end
 function make_interval(::Type{Float64}, x::Float64)
     isinf(x) && return Interval(x)
-    make_interval(Float64, rationalize(x))  # NB: converts a float to a rational -- could be slow
+    split_interval_string(Float64, string(x))
+end
+function make_interval(::Type{Float64}, x::Integer)
+    a = with_bigfloat_precision(53) do
+        make_interval(BigFloat, x)
+    end
+    float(a)
+end
+make_interval(::Type{Float64}, x::BigFloat) = @thin_round(Float64, convert(Float64, x))
+function make_interval(::Type{Float64}, x::Interval)
+    a = make_interval(Float64, x.lo)
+    b = make_interval(Float64, x.hi)
+    Interval(a.lo, b.hi)
 end
 
-make_interval(::Type{Float64}, x::BigInt)    =  @thin_round(BigFloat, convert(Float64, x))
-make_interval(::Type{Float64}, x::BigFloat)  =  @thin_round(BigFloat, convert(Float64, x))
-make_interval(::Type{Float64}, x::Interval)  =  @round(BigFloat, convert(Float64, x.lo), convert(Float64, x.hi)) # NB: BigFloat to Float64 conversion uses *BigFloat* rounding mode
 
 
+# make_interval for Rational intervals
```

Comment:
Before, `transform` was handling everything, and if I recall correctly, a floating point was rationalized, and then the division of (integer) intervals was implemented. I simply maintained the possibility of working with `Interval{Rational}`. 
I think one example is the calculation of π (`calc_pi5`) in the tests.